From b4188f6b8ddaf99482365bc31464e39ddd67cc39 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Thu, 2 Mar 2017 20:39:29 +0800
Subject: [PATCH 1746/5242] MLK-14314-8 dma: pxp: remove meaningless
 assignments in probe()

commit  3aa83bf342c9a47779b038c4dbcec21700a48a50 from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove some assignments which have no real effects for
V2 and V3 PXP drivers, since using devm_kzalloc() to
make the data to be zero.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit a6d6e4d2b6b21427c529a33a7d47ddc918b19eb7)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/dma/pxp/pxp_dma_v2.c |    3 ---
 drivers/dma/pxp/pxp_dma_v3.c |    3 ---
 2 files changed, 6 deletions(-)

diff --git a/drivers/dma/pxp/pxp_dma_v2.c b/drivers/dma/pxp/pxp_dma_v2.c
index a4ee04b..8ab9e29 100644
--- a/drivers/dma/pxp/pxp_dma_v2.c
+++ b/drivers/dma/pxp/pxp_dma_v2.c
@@ -1709,9 +1709,6 @@ static int pxp_probe(struct platform_device *pdev)
 	platform_set_drvdata(pdev, pxp);
 	pxp->irq = irq;
 
-	pxp->pxp_ongoing = 0;
-	pxp->lut_state = 0;
-
 	spin_lock_init(&pxp->lock);
 	mutex_init(&pxp->clk_mutex);
 
diff --git a/drivers/dma/pxp/pxp_dma_v3.c b/drivers/dma/pxp/pxp_dma_v3.c
index 229616e..f489881 100644
--- a/drivers/dma/pxp/pxp_dma_v3.c
+++ b/drivers/dma/pxp/pxp_dma_v3.c
@@ -5187,9 +5187,6 @@ static int pxp_probe(struct platform_device *pdev)
 	platform_set_drvdata(pdev, pxp);
 	pxp->irq = irq;
 
-	pxp->pxp_ongoing = 0;
-	pxp->lut_state = 0;
-
 	spin_lock_init(&pxp->lock);
 	mutex_init(&pxp->clk_mutex);
 
-- 
1.7.9.5

